Standalone_GCBM/tools/CompileGCBMSpatialOutput/create_tiffs.bat

```diff
@@ -2,10 +2,10 @@
 
 REM *************************** USER CONFIGURATION ***************************
 REM Set simulation start year.
-set SIMULATION_START_YEAR=2010
+set SIMULATION_START_YEAR=1900
 
 REM Set Python path - change this to your Python installation directory.
-set GCBM_PYTHON=C:\Python37
+set GCBM_PYTHON=C:\Develop\Python\Python37

```

Comment:
Can you use an environmental variable for this?

Reply:
I refactored all the python paths in bat files in [5296555](https://github.com/moja-global/GCBM.Belize/pull/14/commits/5296555507806e7e63da067c2fbffb51a8e67908)